webviz_subsurface/plugins/_pvt_plot/_views/_pvt/_settings/_view_settings.py

```diff
@@ -0,0 +1,49 @@
+from typing import Dict, List
+
+import webviz_core_components as wcc
+from dash.development.base_component import Component
+from webviz_config.utils import StrEnum
+from webviz_config.webviz_plugin_subclasses import SettingsGroupABC
+
+
+class ViewSettings(SettingsGroupABC):
+    class Ids(StrEnum):
+        SHOW_PLOTS = "show-plots"
+
+    def __init__(self) -> None:
+        super().__init__("Show Plots")
+
+        self.plot_settings = [
+            "Formation Volume Factor",
+            "Viscosity",
+            "Density",
+            "Gas/Oil Ratio (Rs)",
+        ]
+
+    @staticmethod
+    def plot_visibility_options(phase: str = "") -> Dict[str, str]:
+        options = {
+            "fvf": "Formation Volume Factor",
+            "viscosity": "Viscosity",
+            "density": "Density",
+            "ratio": "Gas/Oil Ratio (Rs)",
+        }
```

Comment:
Consider making a `StrEnum` definition?

Reply:
Not really possible since this is a flexible structure (names are changing depending on the phase).